includes/deprecated-functions.php

```diff
@@ -1158,199 +1158,6 @@ function edd_record_status_change( $payment_id, $new_status, $old_status ) {
 	edd_insert_payment_note( $payment_id, $status_change );
 }
 
-/**
```

Comment:
@LisaCee I think in this last merge that some hook deprecations were lost. These should should still be in this PR and updated, please:
- `edd_before_order_history`
- `edd_order_history_row_start`
- `edd_order_history_row_end`
- `edd_after_order_history`

Reply:
Sorry, I missed this comment.